lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
```ruby
(byebug) zipfile.glob("**/**").select(&:directory?).count
0
(byebug) zipfile.glob("**/**").select(&:file?).count
381
(byebug) zipfile.glob("**/**").count
381
```
That is on a zip with several directories containing files. So I'm curious to know how you are getting directories with this command?